src/motor_control.c

```diff
@@ -51,12 +57,14 @@ static inline void reset_tone(MotorControl *mc) {
     mc->tone_high = 0;
 }
 
-void motor_control_apply(MotorControl *mc, float abs_erpm, RunState state, float time) {
-    if (mc->parking_brake_mode == PARKING_BRAKE_ALWAYS ||
-        (mc->parking_brake_mode == PARKING_BRAKE_IDLE && state != STATE_RUNNING && abs_erpm < 50)) {
-        mc->parking_brake_active = true;
-    } else if (mc->parking_brake_mode == PARKING_BRAKE_NEVER || state == STATE_RUNNING) {
-        mc->parking_brake_active = false;
+void motor_control_apply(MotorControl *mc, const MotorData *md, RunState state, float time) {
+    if (mc->parking_brake_mode == PARKING_BRAKE_IDLE) {
+        if (state != STATE_RUNNING &&
+            (md->abs_erpm_smooth < 50 || md->last_erpm_sign != md->erpm_sign)) {
```

Comment:
Is `50` an approximation?

Should this be variable/configurable eventually?

Reply:
> if you have arguments why it should be different let's hear them

I'm not arguing, or suggesting – I'm asking, while reviewing/observing.

> this isn't part of my PR

You're touching this line-of-code and suggesting a change to the surrounding behavior. While we're all here now, look at it, consider it, understand whether `50` is a problem.

It stands out to me as a code-smell that may be contributing to the parking brake feature not behaving as intended, because `50` erpm is unlikely to just be the magic value that makes it work perfectly.

I'm not trying to block your progress or be a jerk, so chill with the defensiveness eh?

Reply:
I put the 50 there. Seemed to work well to me. This value will not be configurable, it's supposed to represent "0" with a small amount of leeway for erpm noise.